alchemist/alchemist-cognitive-agents/src/main/kotlin/it/unibo/alchemist/model/implementations/actions/navigationstrategies/DestinationReaching.kt

```diff
@@ -26,11 +26,11 @@ import it.unibo.alchemist.model.interfaces.geometry.euclidean2d.graph.Euclidean2
  * [KnownDestinationReaching] and [GoalOrientedExploring].
  *
  * @param T the concentration type.
- * @param N the type of landmarks of the pedestrian's cognitive map.
- * @param E the type of edges of the pedestrian's cognitive map.
+ * @param L the type of landmarks of the pedestrian's cognitive map.
+ * @param R the type of edges of the pedestrian's cognitive map.
```

Comment:
Why R for "edges"?

Reply:
The type of `R`elations between landmarks. It's a bit forced, but allows to reuse `N, E` and get rid of `M, F`. They were good, but `F` was used for the type of shape factory in some classes and for that of edges in others.. everything's a bit clearer to me now that `F` is used exclusively for the former.

Reply:
Add a note, something like:
`R the type of edges of the pedestrian's cognitive map, representing the **R**elations between landmarks`

Reply:
In the comment i mean

Reply:
There was already a comment like that in the `OrientingAgent` interface. I added it in every class using that type parameter.